packages/SwingSet/src/kernel/vatManager/deliver.js

```diff
@@ -51,16 +52,16 @@ export function makeDeliver(tools, dispatch) {
     await runAndWait(() => dispatch[dispatchOp](...dispatchArgs), errmsg);
     stopGlobalMeter();
 
-    let status = ['ok'];
+    let status = ['ok', null, null];
     // refill this vat's meter, if any, accumulating its usage for stats
     if (meterRecord) {
       // note that refill() won't actually refill an exhausted meter
-      const used = meterRecord.refill();
+      status[2] = { ...meterRecord.refill(), meterType: METER_TYPE };
```

Comment:
let's keep that `const used=`, it's an extra line that's technically redundant, but it serves to document the fact that we expect `meterRecord.refill()` to return the previous usage, which is not otherwise obvious:

```js
const used = meterRecord.refill();
status[2] = { ...used, meterType: METER_TYPE };
```

(also I'd be tempted to skip the magic-constant name `METER_TYPE`, I guess I usually read those as suggesting that you could just change the constant in one place and everything would be ok, but in fact the two things that must change at the same time are the `meterRecord.refill()` call and the `'tame-metering-1'` value, and having those two things on different lines makes it more likely we'll mess up and change one without the other)

Reply:
I've now inlined the `meterType: 'whatever'` code where the meterUsage structure is actually constructed, along with a `TODO` there to update it if the structure changes.